### ISSUE TITLE
Unused MCK pin setting

### ISSUE THREAD

**Issue by jdthorpe** [feelfreelinux/euphonium#62]
[This line](https://github.com/feelfreelinux/euphonium/blob/master/euphonium/scripts/esp32/base_driver.be#L16) means that the `mck`/`sck` pin is unset even when the user selects an MCK pin in the Driver Plugin UI.   (at least for DACs that don't override the `get_i2s)pins()` method from `base_driver.be`).  I would either drop if from the UI, set to a `select_field` with just one option ("0", "Unused", or "GND")  or allow the user's selection to be passed through the DAC's configuration by default.

In my case, that I connected my DAC and and the result of the undefined / floating voltage on the MCK pin gave me infuriating pops / clicks for the first couple weeks until I discovered that the MCK pin on the DAC should just be connected to ground.  This is pretty confusing and probably leads some new users to (incorrectly!) assume that this project isn't great.

**Comment by feelfreelinux** [comment]
Good point. MCLK does have to actually be routed in DACs. This is actually a bug, as it should be reflected by default in the i2s mclk_io setting.
